jbpm-wb-human-tasks/jbpm-wb-human-tasks-client/src/main/java/org/jbpm/workbench/ht/client/editors/taskslist/TaskAdminListViewImpl.java

```diff
@@ -16,15 +16,46 @@
 package org.jbpm.workbench.ht.client.editors.taskslist;
 
 import javax.enterprise.context.Dependent;
+import javax.inject.Inject;
 
-import static org.jbpm.workbench.ht.model.TaskDataSetConstants.HUMAN_TASKS_WITH_ADMIN_DATASET;
+import org.jboss.errai.ioc.client.api.ManagedInstance;
+import org.jbpm.workbench.common.client.list.ExtendedPagedTable;
+import org.jbpm.workbench.common.client.util.DateUtils;
+import org.jbpm.workbench.common.client.util.GenericErrorSummaryCountCell;
+import org.jbpm.workbench.ht.model.TaskSummary;
+import org.uberfire.ext.services.shared.preferences.GridColumnPreference;
+import org.uberfire.ext.widgets.table.client.ColumnMeta;
+
+import com.google.gwt.dom.client.Style;
+import com.google.gwt.user.cellview.client.Column;
+
+import static org.jbpm.workbench.ht.model.TaskDataSetConstants.*;
+import static org.jbpm.workbench.pr.model.ProcessInstanceDataSetConstants.COLUMN_ERROR_COUNT;
+
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.List;
 
 @Dependent
 public class TaskAdminListViewImpl extends AbstractTaskListView<TaskAdminListPresenter> {
 
     private static final String DATA_SET_TASK_LIST_PREFIX = "DataSetTaskAdminGrid";
     protected static final String TAB_ADMIN = DATA_SET_TASK_LIST_PREFIX + "_0";
+    
+    @Inject
+    private ManagedInstance<GenericErrorSummaryCountCell> popoverCellInstance;
+
 
+    @Override
+    public List<String> getInitColumns() {
```

Comment:
because we now have two distinct implementations, _getInitColumns_ should also be removed from _AbstractTaskListView_ and added to _TaskListViewImpl_